### ISSUE TITLE
User profile custom field labels cannot be disabled

### ISSUE THREAD

**Issue by bascherz** [joomla/joomla-cms#28954]
### Steps to reproduce the issue
1. Add a custom field to a user profile. Any type will do.
2. In the custom field, specify Hide for Show Label.
3. Publish the field.
4. View your profile.

### Expected result
Label should be hidden as specified.

### Actual result

Label is visible.

### System information (as much as possible)

Joomla 3.9.18

### Additional comments

Kind of surprised to be finding this so late in the 3.9 game.

**Comment by ReLater** [comment]
Confirmed, back-end, front-end, Joomla 3.9.19-dev, current staging.

**Comment by brianteeman** [comment]
Thank you for raising this issue.

Joomla 3 is now in security only mode with no further bug fixes or new features.

As this issue doesn't relate to Joomla 4 it will now been closed.

If we are mistaken and this does apply to Joomla 4 please open a new issue (and reference this one if you wish) with updated details for testing in Joomla 4.
cc @zero-24

**Comment by HLeithner** [comment]
@brianteeman can you please stop this posts if you didn't checked if this issue is still valid!
@alikon if you close anything please check if it should really be closed.

**Comment by brianteeman** [comment]
Of course I checked but mistakes can happen.

Clearly the work is not appreciated so will stop doing the work that the various teams say they are doing but really just polish their badges and look at the fluff in their belly buttons. If its not on the first page no on seems to care. Lost track of the number of issues I have seen that are stuck waiting on you.

**Comment by HLeithner** [comment]
Of course your work is appreciated because JBS seems to lack the man power. 

If you find one please ping me again, it's hard for me too keep track in github issues.

As you have seen I only picked 2 random closed issues and both are still valid. That's the reason for my comment.

**Comment by brianteeman** [comment]
and you were wrong 50%

**Comment by alikon** [comment]
@HLeithner i've checked as normally i do
your false allegations surely don't help at all

**Comment by brianteeman** [comment]
Reviewing this again this issue is the same as https://github.com/joomla/joomla-cms/issues/28954 - as that one has some extra context I am closing this one as a duplicate
